README.md

```diff
@@ -3,8 +3,8 @@
 </div>
 <br>
 <p align="center">
-  <a href="https://maven-badges.herokuapp.com/maven-central/com.mapbox.mapboxsdk/mapbox-android-navigation">
-    <img src="https://maven-badges.herokuapp.com/maven-central/com.mapbox.mapboxsdk/mapbox-android-navigation/badge.svg"
+  <a href="https://bintray.com/mapbox/mapbox/com.mapbox.navigation%3Acore/_latestVersion">
```

Comment:
This is referencing to the `mapbox` Bintray public repo and showing `1.0.0-beta.1` release. Wondering how we could get badges using Downloads API 🤔

Reply:
Yea, meant to mention this when I opened the pr. The badge is showing the incorrect version. I guess just remove it for now.